src/test/kotlin/app/infinityflow/keycloak/events/tests/TemporalTest.kt

```diff
@@ -0,0 +1,69 @@
+package app.infinityflow.keycloak.events.tests
+
+import app.infinityflow.keycloak.events.TemporalEventListenerProviderFactory
+import org.keycloak.events.Event
+import org.keycloak.events.EventType
+import org.keycloak.events.admin.AdminEvent
+import org.keycloak.events.admin.AuthDetails
+import org.keycloak.events.admin.OperationType
+import org.keycloak.events.admin.ResourceType
+import java.util.Date
+import java.util.UUID
+import kotlin.test.Test
+
+class TemporalTest {
```

Comment:
i didn't see anything about tests in the pipeline. Will these be run?

Reply:
In general it should as maven runs them in the build lifecycle

If you want to skip tests you would have to run -DskipTests=true